### PR TITLE
Add support for named enums

### PR DESCRIPTION
**Pull request by mfferreira**

### CODE REVIEW

File: src/ques-mapper.ts [hunk 1]
```diff
@@ -30,47 +31,49 @@ export const FhirJsonForm = (
...
         ALL_PROPERTIES[groupProperty]['properties'][
           myProperty
-        ] = ProcessQuestionnaireItem(groupItem);
-        if (GetOptions(groupItem) !== '') {
-          ALL_PROPERTIES[groupProperty]['properties'][myProperty][
-            'enum'
-          ] = GetOptions(groupItem);
-        }
+        ] = GetItemProperties(item);
+
```
Comment (dermatologist) [inline src/ques-mapper.ts:55]: Great. Thanks for cleaning up

File: src/ques-mapper.ts [hunk 2]
```diff
@@ -167,25 +198,25 @@ const GetControlType = (item: R4.IQuestionnaire_Item) => {
...
+  switch (item.type) {
+    case R4.Questionnaire_ItemTypeKind._date:
+      return 'item.answer[0].valueDate'
+    case R4.Questionnaire_ItemTypeKind._time:
+      return 'item.answer[0].valueTime'
+    case R4.Questionnaire_ItemTypeKind._dateTime:
+      return 'item.answer[0].valueDateTime'
+    case R4.Questionnaire_ItemTypeKind._integer:
+      return 'item.answer[0].valueInteger'
+    case R4.Questionnaire_ItemTypeKind._decimal:
```
Comment (dermatologist) [inline src/ques-mapper.ts:210]: Thanks. I keep forgetting that js has case :)

File: src/resp-mapper.ts [hunk 3]
```diff
@@ -1,33 +1,39 @@
...
+import FhirForm from './fhirForm';
+
+export const supportedValueTypes = [
+  'valueString',
+  'valueBoolean',
+  'valueDate',
+  'valueInteger',
+  'valueDecimal',
+  'valueCoding'
+]
```
Comment (dermatologist) [inline src/resp-mapper.ts:13]: Thank you for adding this.

**Review by dermatologist — COMMENTED** [review]
The tests have some issues. Will fix it.

**Comment by mfferreira** [comment]
Will the version be bumped automatically?

**Comment by dermatologist** [comment]
@mfferreira  Just bumped the version to 0.7.0 on npm. Will automate it using GH actions. Thanks again!

**Comment by mfferreira** [comment]
> @mfferreira Just bumped the version to 0.7.0 on npm. Will automate it using GH actions. Thanks again!

Awesome! You're welcome.

I'll possibly open more soon 😏

**Pull request merged**